portal_pages/models.py

```diff
@@ -259,6 +370,13 @@ class CaseStudyPage(Page):
         index.SearchField('summary'),
     )
 
+    marketplace_entry = models.ForeignKey(
+        'portal_pages.MarketplaceEntryPage',
+        null=False,
+        blank=False,
+        related_name="+"
+    )
+
```

Comment:
I think this needs to allow null/blank and also allow for Django to created the backwards relation so we can get from a marketplace entry to its associated case studies (the related_anme="+" is going to prevent that). I can't run the migrations currently since I was experimenting with creating case studies but have no marketplace entry with pk=1. (Making it non-null would also imply info submitters would need to create a marketplace entry before creating a case study...which might actually be desirable, but I'm not sure we've discussed that level of detail yet.)